test/e2e/nstemplatetier_test.go

```diff
@@ -153,54 +152,50 @@ func TestUpdateNSTemplateTier(t *testing.T) {
-	// second group of users: the "cookie lovers"
-	cookieSyncIndexes := setupAccounts(t, awaitilities, "cookie", "cookielover%02d", memberAwait, count)
-
-	// setup chocolate tier to be used for creating spaces without usersignups
-	spaces := setupSpaces(t, awaitilities, "chocolate", "chocolatelover%02d", memberAwait, count)
-
-	cheesecakeSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cheesecakeSyncIndexes, "cheesecake", "base", "base", true)
-	cookieSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cookieSyncIndexes, "cookie", "base", "base", true)
-	verifyResourceUpdatesForSpaces(t, awaitilities, memberAwait, spaces, "chocolate", "base", "base")
-
-	// when updating the "cheesecakeTier" tier with the "advanced" template refs for namespaces (ie, same number of namespaces) but keep the ClusterResources refs
-	updateTemplateTier(t, hostAwait, "cheesecake", "advanced", "")
-	// and when updating the "cookie" tier with the "baseextendedidling" template refs (ie, same number of namespaces)
-	updateTemplateTier(t, hostAwait, "cookie", "baseextendedidling", "baseextendedidling")
-	// and when updating the "chocolate" tier to the "advanced" template for Namespace refs
-	updateTemplateTier(t, hostAwait, "chocolate", "advanced", "")
-
-	// then
-	cheesecakeSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cheesecakeSyncIndexes, "cheesecake", "advanced", "base", false)
-	cookieSyncIndexes = verifyResourceUpdatesForUserSignups(t, awaitilities, memberAwait, cookieSyncIndexes, "cookie", "baseextendedidling", "baseextendedidling", false)
-	verifyResourceUpdatesForSpaces(t, awaitilities, memberAwait, spaces, "chocolate", "advanced", "base")
+	WaitUntilBaseNSTemplateTierIsUpdated(t, hostAwait)
+	baseTier, err := hostAwait.WaitForNSTemplateTier("base")
+	require.NoError(t, err)
 
-	// when updating the "cheesecakeTier" tier with the "advanced" template refs for ClusterResources but keep the Namespaces refs
-	updateTemplateTier(t, hostAwait, "cheesecake", "", "advanced")
-	// and when updating the "cookie" tier to the "base" template refs
-	updateTemplateTier(t, hostAwait, "cookie", "base", "base")
-	// and when updating the "chocolate" tier to the "advanced" template for Namespace refs
-	updateTemplateTier(t, hostAwait, "chocolate", "", "advanced")
+	// create new NSTemplateTiers (derived from `base`)
+	cheesecakeTier := CreateNSTemplateTier(t, hostAwait, "cheesecake", baseTier)
+	cookieTier := CreateNSTemplateTier(t, hostAwait, "cookie", baseTier)
+	chocolateTier := CreateNSTemplateTier(t, hostAwait, "chocolate", baseTier)
+
+	// first group of users: the "cheesecake users"
+	cheesecakeSyncIndexes := setupAccounts(t, awaitilities, cheesecakeTier, "cheesecakeuser%02d", memberAwait, count)
+	// second group of users: the "cookie users"
+	cookieSyncIndexes := setupAccounts(t, awaitilities, cookieTier, "cookieuser%02d", memberAwait, count)
```

Comment:
I guess that we don't need the syncIndexes anymore

Reply:
I don't understand your remark

Reply:
My point was that the syncIndexes were used for checking that there was an update of the MURs, but we don't use MURs for updating the namespaces anymore. So checking the syncIndexes here is not needed.

Anyway, if it works then you can keep it as it is - it can be dropped in one of the following cleanup PRs :+1:

Reply:
yes, that thing was not in the scope of the refactoring I wanted to propose. If these `syncIndexes` are not needed anymore, they should have been removed in a previous PR (when their usage was effectively dropped)

Reply:
I planned to remove them in an upcoming PR